ansible/roles/metrics/files/process-cloudflare/process-cloudflare.js

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env node
+
+const { pipeline, Transform } = require('stream')
+const split2 = require('split2')
+const strftime = require('strftime').timezone(0)
+const {Storage} = require('@google-cloud/storage');
+
+const storage = new Storage({keyFilename: "metrics-processor-service-key.json"});
+
+const jsonStream = new Transform({
+  readableObjectMode: true,
+  transform (chunk, encoding, callback) {
+    try {
+      this.push(JSON.parse(chunk.toString()))
+      callback()
+    } catch (e) {
+      callback(e)
```

Comment:
Let's remove this `e` from here. It'll end the stream but that may not be what we want. Can't parse a line? Ignore.